data/json/bionics.json

```diff
@@ -778,6 +779,7 @@
     "name": "Air Filtration System",
     "description": "Surgically implanted in your trachea is an advanced filtration system.  If toxins, or airborne diseases find their way into your windpipe, the filter will attempt to remove them.",
     "occupied_bodyparts": [ [ "TORSO", 4 ], [ "MOUTH", 2 ] ],
+    "env_protec": 7,
```

Comment:
Not sure how it works, but wouldn't it result in giving environmental protection to torso as well? I don't think this bionic reducing acid damage over torso is intended.

Reply:
Ah... True it would. I need to change some stuff then

Reply:
Now It specifies both the value and the protected body parts